ert/data/_record.py

```diff
@@ -365,7 +362,7 @@ async def _load_blob_record(self) -> BlobRecord:
             data = await f.read()
         return BlobRecord(data=data)
 
-    async def dump(self, location: Path) -> None:
+    async def dump(self, location: Path, mime: str) -> None:
         if not self.is_transmitted():
             raise RuntimeError("cannot dump untransmitted record")
         await _copy(self._uri, str(location))
```

Comment:
Hmm, this assumes that the internal mime type used for the shared disk storage is the same as the mime we want to dump the record as. (Holds true for now since we only allow json). Don't think we need to address it in this PR, but we should make an issue :thinking:

Reply:
https://github.com/equinor/ert/issues/1964